include/deal.II/base/symmetric_tensor.h

```diff
@@ -1217,14 +1217,14 @@ namespace internal
       case 2:
         return (data[0] * sdata[0] +
                 data[1] * sdata[1] +
-                Number(2.) * data[2] * sdata[2]);
```

Comment:
@kronbichler @bangerth what about wrapping `2.0` in a `inline` function, see https://github.com/dealii/dealii/pull/3967#issuecomment-279236132 ?